Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -922,6 +1334,35 @@ def incident_attachments(client, incident_id):
     return response
 
 
+def upload_incident_attachment(client: SimpleClient, incident_id: str, entry_id: str, tag_to_ibm: str):
+    """
+        Uploads a file from XSOAR to the IBM QRadar SOAR incident with ID `incident_id`.
+    """
+    try:
+        file_path_obj = demisto.getFilePath(entry_id)
+    except ValueError:
+        return CommandResults(
+            entry_type=EntryType.ERROR,
+            readable_output=f"Could not find a file with entry ID: {entry_id}",
+        )
+
+    file_path, file_name = file_path_obj.get("path"), file_path_obj.get("name")
+    if '.' in file_name:
+        tagged_file_name, extension = file_name.split('.')[0], file_name.split('.')[1]
+        file_name = f'{tagged_file_name}_{tag_to_ibm}.{extension}'
+    response = client.post_attachment(
+        uri=f"/incidents/{incident_id}/attachments",
+        filepath=file_path,
+        filename=file_name,
+    )
+    demisto.debug(f"upload_incident_attachment_command {response=}")
```

Comment:
no return statement.

Reply:
there shouldn't necessarily be, it is not a command function. just executing the file upload.